packages/cli/src/CommandsLogic.ts

```diff
@@ -194,7 +201,45 @@ export default class CommandsLogic {
 
       if (owner !== constants.ZERO_ADDRESS && !isSameAddress(owner, options.from)) {
         // throw new Error(`Already owned by ${owner}, our account=${options.from}`)
-        throw new Error(`Already owned by ${owner}, our account=`)
+        throw new Error(`Already owned by ${owner}, our account=${options.from}`)
+      }
+
+      const bal = await this.contractInteractor.getBalance(relayAddress)
+      if (toBN(bal).gt(toBN(options.funds.toString()))) {
+        console.log('Relayer already funded')
+      } else {
+        console.log('Funding relayer')
+
+        const _fundTx = await this.web3.eth.sendTransaction({
+          from: options.from,
+          to: relayAddress,
+          value: options.funds,
+          gas: 1e6,
+          gasPrice: options.gasPrice
+        })
+        const fundTx = _fundTx as TransactionReceipt
+        if (fundTx.transactionHash == null) {
+          return {
+            success: false,
+            error: `Fund transaction reverted: ${JSON.stringify(_fundTx)}`
+          }
+        }
+        transactions.push(fundTx.transactionHash)
+      }
+
+      if (owner === constants.ZERO_ADDRESS) {
+        let i = 0
+        while (true) {
+          await sleep(options.sleep)
+          const newStakeInfo = await stakeManager.getStakeInfo(relayAddress)
+          if (isSameAddress(newStakeInfo.owner, options.from)) {
```

Comment:
should test for non-zero, non-from.
currently if it is configured with wrong address, you'll have to wait the entire timeout (and then crash on the next owner transaction)